tools/Get-NuGetTool.ps1

```diff
@@ -0,0 +1,15 @@
+<#
+.SYNOPSIS
+    Downloads the NuGet.exe tool and returns the path to it.
+#>
+
+$binaryToolsPath = "$PSScriptRoot\..\obj\tools"
+if (!(Test-Path $binaryToolsPath)) { $null = mkdir $binaryToolsPath }
+$nugetPath = "$binaryToolsPath\nuget.exe"
+if (!(Test-Path $nugetPath)) {
+    $NuGetVersion = "3.4.4"
```

Comment:
@GeertvanHorrik like most things, I would suggest pinning to a specific version, as is done here, that way there are no surprises when the build just suddenly stops working.  If/when we need to update to a new version of Nuget then we can do this, bu changing the version number that we are currently pinned to.